roswallp.txt

```diff
@@ -0,0 +1,16 @@
+[Section]
+Name = ReactOS Standard Wallpapers
+Version = 1
+License = CC BY
+Description = The standard wallpaper collection for ReactOS
+Category = 16
+URLSite = https://github.com/reactos/reactos/
+URLDownload = https://github.com/katahiromz/ReactOSWallpapers/releases/download/v1/ReactOSWallpapers-v1.zip
```

Comment:
Do you want me to create a `Wallpapers` repository?

Reply:
Yes if you can.

Reply:
https://github.com/reactos/wallpapers

I'll leave it up to you to fill in the readme and define a directory structure and whatnot.

If you need any changes you can't make, let me know.

Reply:
https://github.com/reactos/wallpapers/pull/1